cli/cmd/compose/pull.go

```diff
@@ -30,8 +30,11 @@ type pullOptions struct {
 	composeOptions
 }
 
-func pullCommand() *cobra.Command {
-	opts := pullOptions{}
+func pullCommand(composeOpts composeOptions) *cobra.Command {
+	opts := pullOptions{
+		composeOptions: composeOpts,
```

Comment:
This also needs to be a reference.

We haven't yet parsed the flags so we're making a copy of an empty struct here. What we want to do is pass a reference so that when the flags are parsed, our struct will have its fields set.